<launch>
  <node pkg="joy2twist" type="joy2twist_node" name="joy2twist">	
  	<!-- Axes -->
  	<param name="axis_linear" value="5" type="int"/>
  	<param name="axis_angular" value="0" type="int"/>
  	<param name="scale_linear" value="1" type="double"/>
  	<param name="scale_angular" value="-0.8" type="double"/>
  </node>
</launch>